docs/node/run-your-node/sentry-node.md

```diff
@@ -225,78 +146,33 @@ Before using this configuration you should collect the following information to
   on the sentry node.
 
 ```yaml
-##
-# Oasis Node Configuration
-#
-# This file's configurations are derived from the command line args found in
-# the root command of the oasis-node binary. For more information execute:
-#
-#     oasis-node --help
-#
-# Settings on the command line that are separated by a dot all belong to the
-# same nested object. So "--foo.bar.baz hello" would translate to:
-#
-#     foo:
-#       bar:
-#         baz: hello
-##
-
-# Set this to where you wish to store node data. The node artifacts
-# should also be located in this directory (for us this is /node/data)
-datadir: /node/data
-
-# Logging.
-#
-# Per-module log levels are defined below. If you prefer just one unified log
-# level, you can use:
-#
-# log:
-#   level: debug
-log:
-  level:
-    # Per-module log levels. Longest prefix match will be taken. Fallback to
-    # "default", if no match.
-    default: debug
-    tendermint: warn
-    tendermint/context: error
-  format: JSON
-  # By default logs are output to stdout. If you're running this in docker keep
-  # the default
-  #file: /var/log/oasis-node.log
-
-# Path to the genesis file for the current version of the network.
-genesis:
-  file: /node/etc/genesis.json
-
-# Worker configuration.
-worker:
-  registration:
-    # In order for the node to register itself the entity.json of the entity
-    # used to provision the node must be available on the node.
-    entity: /node/etc/entity.json
-  sentry:
-    address:
-      - "{{ sentry_node_grpc_public_key }}@{{ sentry_node_private_ip }}:9009"
-
-# Consensus backend.
+mode: client
```

Comment:
There were previously two configs, one for the sentry node, and one for the "normal" node, that is connecting to the sentry. Seems that one of the configs is missing now?

Reply:
Yeah, you are right. Added both configs and re-added comments.